elastic/kibana
# Remove transaction type filter in errors table

Closes https://github.com/elastic/kibana/issues/113612

### What was done

- Remove the `transactionType` from the queries to the errors table in the service overview.

### How to test

- On a service with different types of transaction select one from the dropdown
- Check that errors table won't change and when navigation to errors page the same errors are shown

BEFORE

Service overview page

![image](https://user-images.githubusercontent.com/31922082/152979989-102c6fb0-47c7-4a30-adef-71e1ed6c8127.png)

Errors page

![image](https://user-images.githubusercontent.com/31922082/152980376-fcaef34e-6286-4bb3-a9b4-13ed279abcd0.png)


AFTER

Service overview page

![image](https://user-images.githubusercontent.com/31922082/152980288-1f969e12-939e-43e6-bbb6-d32aa25ec477.png)
